src/main/scala/rocket/ICache.scala

```diff
@@ -110,92 +171,271 @@ class ICachePerfEvents extends Bundle {
   val acquire = Bool()
 }
 
+/** IO from CPU to ICache. */
 class ICacheBundle(val outer: ICache) extends CoreBundle()(outer.p) {
+  /** first cycle requested from CPU. */
   val req = Decoupled(new ICacheReq).flip
-  val s1_paddr = UInt(INPUT, paddrBits) // delayed one cycle w.r.t. req
-  val s2_vaddr = UInt(INPUT, vaddrBits) // delayed two cycles w.r.t. req
-  val s1_kill = Bool(INPUT) // delayed one cycle w.r.t. req
-  val s2_kill = Bool(INPUT) // delayed two cycles; prevents I$ miss emission
-  val s2_cacheable = Bool(INPUT) // should L2 cache line on a miss?
```

Comment:
This line should not be removed.

Reply:
I think this is just formatting. The code is the same

Reply:
I actually prefer the old format for this level of commentary, its much more concise.

Reply:
Agreed.